rules/readme-has-general-usage.js

```diff
@@ -0,0 +1,42 @@
+let rule = require("unified-lint-rule")
+let walk = require("unist-util-visit")
+let { join } = require("path")
+let { existsSync, readFileSync } = require("fs")
+
+/**
+ * Has a link to general component usage information:
+ * https://origami.ft.com/docs/components
+ *
+ * @param {import("mdast").Root} tree
+ */
+function readmeHasGeneralUsage(tree, file) {
+    if (file.basename.toLowerCase() != "readme.md") {
+        return
+    }
+
+    const generalUsageLink = 'https://origami.ft.com/docs/components';
```

Comment:
Nit: I think this project uses tabs for spacing and double quotes instead of single quotes for strings